src/app/pages/charging-stations/charging-limit/charging-profile-limit/charging-station-charging-profile-limit-schedule-table-data-source.ts

```diff
@@ -93,10 +93,9 @@ export class ChargingStationChargingProfileLimitScheduleTableDataSource extends
 
   public setChargingProfileSchedule(schedules: Schedule[]) {
     this.schedules = schedules;
-    this.getManualDataChangeSubject().next();
```

Comment:
The list of charging profile is no longer displayed.
I'll put it back.

Reply:
It's not refreshed because socket.io is not listening to changes on the right collection yet in that PR.

Reply:
> It's not refreshed because socket.io is not listening to changes on the right collection yet in that PR.

No, the call to the DB is already done, these data sources are disconnected from the DB thus the 'manual refresh' when assigned new values.